src/test/scala/chiselTests/package.scala

```diff
@@ -0,0 +1,7 @@
+import chisel3.core.LegacyModule
+
+package object chiselTests {
+  // We don't want firrtl complaining about "not fully initialized" connections.
+  val implicitInvalidateOptions = chisel3.core.ExplicitCompileOptions.Strict.copy(explicitInvalidate = false)
+  abstract class ImplicitInvalidateModule extends LegacyModule()(implicitInvalidateOptions)
```

Comment:
If this is breaking enough tests that you don't want to fix all the cases correctly and need to create an abstraction to patch around existing issues, you might want to rethink the API